src/test/java/pages/LogInPage.java

```diff
@@ -13,7 +13,7 @@ public LogInPage(WebDriver driver) {
         this.driver = driver;
     }
 
-    @FindBy(xpath = "//*[text() = 'Sign In']")
+    @FindBy(xpath = "//*[@id=\"root\"]/main/div/div[1]/div/span/h6")
```

Comment:
It will be better to write xpath in such way @FindBy(xpath = "//*[text() = 'Sign In']"), because it is more correct. I see, that you used possibility Copy - Copy XPath in Chrome, but it is not so good, the previous option was better.